### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by david63011**
Make sure that there are [no linter errors]:
Linter should be set up correctly for this repository.
Linter result should be green.
There should be no custom changes to linter config.
Make sure that you used correct git flow 
README file should be descriptive.
Commit messages should be meaningful.

### CODE REVIEW

**Review by Qoosim — CHANGES_REQUESTED** [review]
Hi @david63011 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ]  has linter error.

linter-error

- [ ] Kindly fix the error and submit for another review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by david63011** [comment]
hey qoosim,
that was for javascript and I don't have any javascript thats why it
failed. it is only css and html I cant change anything

**Comment by david63011** [comment]
there wasnt an error I can change

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @david63011, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project that has failing linters, Kindly make sure your linters are all passing before submitting the PR.

This activity requires you to configure linters for HTML&CSS excluding javascript. Kindly remove the eslint workflow from your linters.yml file 👍🏼 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @david63011,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

### ❌ Invalid Code Review Request
You have submitted a project that has no linters, Kindly make sure your linters are all working before submitting the PR.

**_kindly follow this [guide ](https://github.com/microverseinc/linters-config/tree/master/html-css)on how to install HTML/CSS Linters 👍🏼_**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Invalid Code Review Request does not count into the code reviews limit.**

File: index2.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          <p class="job-info">Developer</p>
+        </div>
+        <div class="information-box">
+          <h1 class="info">I can help you build a product,</h1>
+          <h1 class="info2">feature or website Look through</h1>
+          <h1 class="info3">some of my work and experience!</h1>
+          <h1 class="info4">If you like what you see and have</h1>
+          <h1 class="info5">a project you need coded</h1>
+          <h1 class="info6">don’t hestiate to contact me.</h1>
+        </div>
```
Comment (mwanawabangona) [inline index2.html:39]: - [ ] `required`
- [ ] its not best practice to break one word into chuncks according to to the [html&Css](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) sematic tags kindly put the pagragraph in a ptag

File: index2.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        <div class="column2">
+          <div class="project-box">
+            <h1 class="project-name1">Multi-Post Stories</h1>
+            <h1 class="project-name-two">Gain+Glory</h1>
+          </div>
+          <div class="languages">
+            <div class="ruby">Ruby on Rails</div>
+            <div class="css">css</div>
+            <div class="js">JavScript</div>
+            <div class="html">HTML</div>
```
Comment (mwanawabangona) [inline index2.html:77]: - [ ] `required`
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag).
- Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.
- [ ] to the [html&Css](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) sematic tags

it should like something like this 
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @david63011,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @david63011  , I'm @MaganaAsiati , your reviewer for this PR  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


clap

## Highlights
- Add a descriptive description in your pull request :heavy_check_mark:👨🏻‍💻
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- check comment under the review
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**